docs/getting-started-with-cosmosdb.md

```diff
@@ -0,0 +1,53 @@
+## Getting Started with Scalar DB on Cosmos DB
```

Comment:
```suggestion
# Getting Started with Scalar DB on Cosmos DB
```

Reply:
Thank you for your suggestion.
The document updated accordingly.